src/magneticField/turbulentField/PlaneWaveTurbulence.cpp

```diff
@@ -169,6 +136,16 @@ PlaneWaveTurbulence::PlaneWaveTurbulence(const TurbulenceSpectrum &spectrum,
 
 		Vector3d kappa =
 		    Vector3d(sintheta * cos(phi), sintheta * sin(phi), costheta);
+		// NOTE: all other variable names match the ones from the TD13 paper.
+		// However, our xi is actually their psi, and their xi isn't used at
+		// all. (Though both can be used for the polarization vector, according
+		// to the paper.) The reason for this discrepancy is that this code
+		// used to be based on the original GJ99 paper, which provided only a
+		// xi vector, and this xi happens to be almost the same as TD13's psi.
+		// Hence, the code kept both the name and the rough shape of the vector,
+		// but is now in disagreement with the TD13 paper. I don't think there's
+		// a good reason to keep it this way, except that I do not want to
+		// be the one who changes it.
```

Comment:
Remove the last sentence "I don't think there's a good reason to keep it this way, except that I do not want to be the one who changes it."